### DISCUSSION TITLE
Flip the order of years shown in Archives

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#186] [Feature requests] [answered]
![image](https://user-images.githubusercontent.com/25685344/188263913-b66b7db4-3a31-4287-b38a-cc0f12eb9ff5.png)
Currently archived seasons are shown with the oldest first putting the more relevant seasons/races to Multiviewer users at the bottom.

**Comment by f1multiviewer** [comment]
Duplicate of #70

**Comment by f1multiviewer** [comment, marked as answer]
Done!
